test-object.json

```diff
@@ -1,8 +0,0 @@
-[
-  {
```

Comment:
Kind of. These files snuck into an earlier PR and aren't necessary, which is why I'm removing them.